sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -34,7 +34,9 @@ class FlinkRunner(portable_runner.PortableRunner):
   def default_job_server(self, options):
     flink_master = options.view_as(FlinkRunnerOptions).flink_master
     if flink_master == '[local]' or sys.version_info < (3, 6):
-      # TOOD(BEAM-8396): Also default to LOOPBACK for [local].
+      portable_options = options.view_as(pipeline_options.PortableOptions)
+      if flink_master == '[local]' and not portable_options.environment_type:
+        portable_options.environment_type == 'LOOPBACK'
```

Comment:
I wonder, what is the effect of this statement?

Reply:
There's a little blurb on loopback here: https://beam.apache.org/roadmap/portability/#sdk-harness-config

I think the main goal of this PR was to prevent local file system issues when using Docker workers (files not being shared between workers, "Where'd my output go?") for people using the Flink runner for the first time.

Reply:
I'm familiar with the loopback environment :), but the statement has no effect, since it is using `==` instead of `=`. I've fixed this in #9918.

Reply:
Ah, totally missed that... Python is too slippery. This is why we need tests :sweat_smile: